cmd/notary/keys.go

```diff
@@ -127,11 +128,7 @@ func (k *keyCommander) GetCommand() *cobra.Command {
 	cmdRotateKey.Flags().BoolVarP(&k.rotateKeyServerManaged, "server-managed", "r",
 		false, "Signing and key management will be handled by the remote server. "+
 			"(no key will be generated or stored locally) "+
```

Comment:
I think it might make sense for this parenthesized segment to go into the last sentence, so that it'd look like this with the current phrasing:
`Signing and key management will be handled by the remote server (no key will be generated or stored locally.  Can only be used for timestamp and snapshot roles, and MUST be used for the timestamp role.`